lms/models/lms_course.py

```diff
@@ -71,7 +71,7 @@ class LMSCourseMembership(CreatedUpdatedMixin, Base):
     )
 
     lms_user_id: Mapped[int] = mapped_column(
-        sa.ForeignKey("lms_course.id", ondelete="cascade"), index=True
```

Comment:
This worked locally because both tables created at the same time, starting in PK `1` allowed to cross-reference both values.